pylzy/lzy/api/v1/workflow.py

```diff
@@ -259,6 +259,13 @@ def __gen_results_uri_in_cache(self, call: 'LzyCall'):
                 entry.data_hash = md5_of_str(entry.storage_uri)
                 self.__filled_entry_ids.add(eid)
 
+        eid = call.exception_id
+        if eid not in self.__filled_entry_ids:
```

Comment:
I'm not sure that we should cache exception

summon @mrMakaronka

Reply:
agree with @imakunin , exceptions should not be cached